modules/storage-persistent-storage-block-volume.adoc

```diff
@@ -48,6 +48,7 @@ endif::openshift-dedicated,openshift-rosa[]
 ifndef::openshift-dedicated,openshift-rosa[]
 |NFS | | |
 |{rh-storage-first} | ✅ | ✅ | ✅
+|SMB CIFS  |  |  | 
```

Comment:
we need to fill up the the blanks here, dynamic & static have been tested by QE
cc @mpatlasov @Phaow

Reply:
Hmm, this section is raw block volume support matrix, so from my understanding it should be all not supported. It seems this section also does not unify the unsupport symbol. I saw in `Access modes
` support we use `-` for unsupported, but this raw block just use empty to indicate unsupported. We'd better unify the doc fomat.
```suggestion
|SMB CIFS  | - | - | -
```

Reply:
I've standardized on checkmarks and blanks. Also, I've filled in the blanks for the block vol table.